blackbox/docs/admin/cluster-management.rst

```diff
@@ -0,0 +1,127 @@
+.. _cluster_management:
+
+==================
+Cluster management
+==================
+
+This document shows you how to manage your CrateDB clusters.
```

Comment:
I think the title, description and placement in the TOC promote this too much as a "general cluster management" tool. 

In day-to-day operations users/admins should almost never have to use this tool. The main use-case is for disaster recovery.

Reply:
you're right

perhaps we should make the following changes:

- make a new top-level section in the reference called "CLI tools"
- "CLI tools" should appear in the navigation where "Running CrateDB" does at the moment
- the "Running CrateDB" doc can be moved to "`crate`" under "CLI tools"
- this doc can be moved from "Cluster management" to "`crate-node`" under "CLI tools"

this mirrors the way Elasticsearch do it (see https://www.elastic.co/guide/en/elasticsearch/reference/current/node-tool.html)

Reply:
we will need to add in some redirections for this restructuring, but that's fine

Reply:
holding off making any other comments for now. I think we should resolve this / perform the changes outlined in my comment above (assuming Jordi agrees) first

Reply:
Not sure if "Running CrateDB" will then still be prominent enough, but otherwise I like the idea.

Reply:
Okay, so next I'll execute all of the above. Once done, I'll ping you, Nomi, for the redirects and a review.